specs/host_list_view.adoc

```diff
@@ -0,0 +1,111 @@
+// vim: tw=79
+
+= Tendrl Definitions: Display list of hosts discovered by Tendrl
+
+Display list of hosts discovered by Tendrl, whether they currently associated with a cluster or not. The list should have functionality for 
+
+* filtering by name, cluster, role and status.
+* sorting by name.
+
+Host list will be sorted by name by default.
+
+== Problem description
+
+Tendrl should show list of hosts discovered by Tendrl along with their details.
+
+The details shown for each host in UI is -
+
+* fqdn
+
+== Use Cases
+
+Users can see the list of host present in Tendrl. They can also filter and sort the list. 
+
+== Proposed change
+
+* Create a view for listing host details.
+* This view will use generate-list component for listing host. The list data will be passed to this component from the controller.
+* Implement filer and sorting functionality.
+* Filter parameter(name/cluster/role/status) should be sent to API to fetch the filtered list of host.
+* Sorting parameter(name and asc/desc) should be sent to API to fetch sorted list of host.
+
+=== Alternatives
+
+Host list can have its own html template instead of using generate-list component. But the code will be redundant as the same functionality and layout is going to be used in all list views.
+
+=== Data model impact:
+
+
+=== Impacted Modules:
+
+==== Tendrl API impact:
+
+
+==== Notifications/Monitoring impact:
```

Comment:
How the live status updates are handled, say host status change or utlization changes?
How the asynchronous notifications from backend are handled updated on the list view?